### ISSUE TITLE
acceptance_tests is faied on Ruby 2.4

### ISSUE THREAD

**Issue by junaruga** [cucumber/cucumber-ruby-wire#6]
On master branch, Fedora 24, ruby 2.4.0, bundler 1.13.7

```
bundle exec rake unit_tests
```
is done successfully.

However

```
bundle exec rake acceptance_tests
...
      +/home/jaruga/git/cucumber-ruby-wire/vendor/bundle/ruby/2.4.0/gems/diff-lcs-1.2.5/lib/diff/lcs/change.rb:117: warning: constant ::Fixnum is deprecated
...
Failing Scenarios:
cucumber features/table_diffing.feature:23 # Scenario: Invoke a step definition tries to diff the table and fails
cucumber features/table_diffing.feature:52 # Scenario: Invoke a step definition tries to diff the table and passes
cucumber features/table_diffing.feature:71 # Scenario: Invoke a step definition which successfully diffs a table but then fails
cucumber features/table_diffing.feature:98 # Scenario: Invoke a step definition which asks for an immediate diff that fails

21 scenarios (4 failed, 17 passed)
```

Maybe the reason is because of below issue. diff-lcs-1.2.5 outputs warnings on Ruby 2.4.
https://github.com/halostatue/diff-lcs/pull/36/files

**Comment by mattwynne** [comment]
@junaruga thanks so much for digging into this. It looks like the diff-lcs issue is fixed in their 1.3 release. Could you try upgrading and see if you can send us a PR?

**Comment by junaruga** [comment]
@mattwynne I succeeded the both tests after `diff-lcs` was upgraded to 1.3.

```
$ ruby -v
ruby 2.4.0p0 (2016-12-24 revision 57164) [x86_64-linux]

$ git pull

$ bundle install --path vendor/bundle

$ bundle list
Gems included by the bundle:
  * aruba (0.14.2)
  * backports (3.6.8)
  * builder (3.2.3)
  * bundler (1.14.3)
  * childprocess (0.5.9)
  * contracts (0.14.0)
  * cucumber (3.0.0.pre.1)
  * cucumber-core (2.0.0)
  * cucumber-wire (0.0.1)
  * diff-lcs (1.3)
  * ffi (1.9.17)
  * gherkin (4.0.0)
  * multi_json (1.12.1)
  * multi_test (0.1.2)
  * rake (12.0.0)
  * rspec (3.5.0)
  * rspec-core (3.5.4)
  * rspec-expectations (3.5.0)
  * rspec-mocks (3.5.0)
  * rspec-support (3.5.0)
  * thor (0.19.4)

$ bundle exec rake unit_tests
...
20 examples, 0 failures

$ bundle exec rake acceptance_tests
...
17 scenarios (17 passed)
99 steps (99 passed)
0m7.954s
```
